code/noRAM/app/src/main/java/com/example/noram/model/Database.java

```diff
@@ -138,7 +132,7 @@ public void downloadPhoto(String photoRef, Consumer<Bitmap> completeFunction) {
                 completeFunction.accept(image);
             };
             executor.execute(decodeRunnable);
-        });
+        }).addOnFailureListener(e -> completeFunction.accept(null));
```

Comment:
Is this to replace the current workaround we have with Event's null-pictures?

Reply:
i dont think this really affects much with the app, but it's mainly just because i thought the behavior of downloadPhoto was a bit inconsistent, since it the download fails, the callback wouldn't ever get called. I figured it made more sense to make sure the callback will always get called, but with null if it failed